test/services/user_update_test.rb

```diff
@@ -216,19 +216,4 @@ class UserUpdateTest < ActionView::TestCase
       UserUpdate.new(affected_user, params).call
     end
   end
-
```

Comment:
Unfortunately, the diff this comment is pointing at now seems to be outdated, presumably because of the rebase against `main` and force-push. @chrislo I don't suppose you can remember what this was referring to, can you?

Reply:
@floehopper - I suspect he's talking about the removal of `UserUpdateTest#with_current` in this commit.

Reply:
Addressed in https://github.com/alphagov/signon/pull/2623/commits/fd9556d405fbf425e652b5bb3421d6c6bdd54340. Marking as resolved.

Reply:
Addressed correctly in https://github.com/alphagov/signon/pull/2623/commits/d6b66486598046fe6a96371888ae86ab03fec982 & https://github.com/alphagov/signon/pull/2623/commits/19af8bcaccee0aed4468187badb2ebc496f53f5c.